### DISCUSSION TITLE
2023년 05월 01일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#49] [General]
# 2023년 05월 01일

LOG 갑자기 오늘따라 눈이 일찍 떠져서 6시에 일어났다;;; 다시 자려고 했는데 잠이 안 왔다.. 일찍 일어난 김에 오랜만에 아침을 만들어서 먹었다. 집에 스팸이 쌓여서 스팸 김치 볶음밥을 해먹었는데 맛있었다. 빨래도 하고 설거지도 하고 오랜만에 …

https://cyma-s.github.io/2023-05-01/

**Comment by kevstevie** [comment]
감기 그만 걸리세요

**Reply by Cyma-s** [reply]
감기 양방향 의존 그만 끊죠?
